llamacpp-server-launcher.py

```
@@ -1067,6 +1092,45 @@ def _setup_advanced_tab(self, parent):
+        # Up/Down buttons for reordering
+        order_buttons_frame = ttk.Frame(gpu_order_frame)
+        order_buttons_frame.pack(side="left", padx=(5, 0))
+
+        self.gpu_order_up_btn = ttk.Button(order_buttons_frame, text="▲ Up", command=self._move_gpu_order_up, width=8)
+        self.gpu_order_up_btn.pack(side="top", pady=(0, 2))
+
+        self.gpu_order_down_btn = ttk.Button(order_buttons_frame, text="▼ Down", command=self._move_gpu_order_down, width=8)
+        self.gpu_order_down_btn.pack(side="top")
+
+        ttk.Label(inner, text="First GPU in list = GPU 0 for tensor split. Affects CUDA_VISIBLE_DEVICES order.", font=("TkSmallCaptionFont"))\
+            .grid(column=3, row=r, sticky="w", padx=5, pady=3); r += 1
```

Ruff (0.15.10)

[error] Multiple statements on one line (semicolon)

(E702)